x-pack/plugins/observability_solution/logs_shared/public/plugin.tsx

```diff
@@ -76,6 +88,14 @@ export class LogsSharedPlugin implements LogsSharedClientPluginClass {
       search: data.search.search,
       uiSettings: settings,
       share,
+      dataViews,
+      fieldFormats,
+      columns: {
+        SummaryColumn: (props: Omit<AllSummaryColumnProps, 'core' | 'share'>) => {
```

Comment:
Can you help me understand what the advantage of binding the dependencies here is over doing that at the usage site?

Reply:
Just so the usage site didn't have to access context etc.

Reply:
right - I guess what made me stumble over this is that this is not exported, but used in a location where `share` is already available anyway, because `dependencies` are being passed around. In that sense it breaks the pattern used in the logs overview. If this was exported on the plugin contract that would have been quite logical.

Anyway, not a big deal, I'll defer to your decision. :innocent:

Reply:
Changed here: https://github.com/elastic/kibana/pull/194867/commits/7b2545e7341b01b2b442c855a982b48920fb0852